packages/kbn-optimizer/package.json

```diff
@@ -45,8 +45,8 @@
   },
   "devDependencies": {
     "@types/compression-webpack-plugin": "^2.0.2",
-    "@types/loader-utils": "^1.1.3",
+    "@types/loader-utils": "^2.0.1",
     "@types/watchpack": "^1.1.5",
-    "@types/webpack": "^4.41.3"
```

Comment:
It looks like master has `"@types/webpack": "^4.41.3"` and `"@types/loader-utils": "^1.1.3",`. Mind aligning with those to keep the branches consistent?